awx/main/utils/formatters.py

```diff
@@ -21,7 +24,81 @@ def format(self, record):
         return logging.Formatter.format(self, record)
 
 
-class LogstashFormatter(LogstashFormatterVersion1):
```

Comment:
Previously, we inherited from `LogstashFormatterVersion1`, which inherited from `LogstashFormatterBase`.

However, `LogstashFormatterVersion1` only defined 1 method, `format`. We overwrote this method entirely.